certdocs/TRAQ-138-SDD.md

```diff
@@ -997,6 +997,23 @@ Reqtraq SHALL provide a command line option to start a local web sever to genera
 - Verification: Test
 - Safety Impact: None
 
+### clang.go
+
+Reqtraq can use libclang to parse the ast of any linked code and obtain code references. This option
+is preferred for a fine-grained code parsing, while ctags is preferred for compatibility with other 
+languages like Go.
+
+#### REQ-TRAQ-SWL-61 Support for libclang backend to parse code
```

Comment:
I feel this deserves more than one LLR. There's some derived functionality in the code which should be specified in low-level requirements (along with rationales), e.g. the skipping of certain elements, the use of command database